duckduckgo/BrowserServicesKit
# Support for TDS options

**Required**:

Task/Issue URL: https://app.asana.com/0/1203790657351911/1204149290597759/f
iOS PR: https://github.com/duckduckgo/iOS/pull/1666
macOS PR: https://github.com/duckduckgo/macos-browser/pull/1149
What kind of version bump will this require?: Patch

CC: @jonathanKingston 

**Description**:
Support for TDS options.

**Steps to test this PR**:
1. Make sure all tests pass.
1. Change tds endpoint in AppURLs to "https://staticcdn.duckduckgo.com/trackerblocking/beta/tds-options.json"
1. Remove the app from the simulator and recompile.
1. Visit https://privacy-test-pages.glitch.me/tracker-site-blocking/index.html
1. Open Site A - inspect with proxyman or some other tool that publisher-company.site is never loaded.
1. Also, there should be an information on Dashboard about it being blocked.
1. Open Site B - we should not block anything.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
